File: 0041-first-missing-positive/0041-first-missing-positive.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var firstMissingPositive = function(nums) {
        let x=1;
          nums.sort((a, b) => a-b);
        for( i=0;i<nums.length; i++) {
        if(nums[i] == x){
            x++;  
        }
         
    }
    return x
};

File: 0041-first-missing-positive/0041-first-missing-positive.cs
public class Solution {
    public int FirstMissingPositive(int[] nums) {
        int x =1;
        Array.Sort(nums);

        for(int i =0 ; i<nums.Length;i++){
            if (nums[i] == x){
                x++;
            }
        }

return x;
    }
}